Formula/sparkctl.rb

```diff
@@ -0,0 +1,26 @@
+class Sparkctl < Formula
+  desc "CLI for interacting with spark on k8s"
+  homepage "https://github.com/GoogleCloudPlatform/spark-on-k8s-operator"
+  url "https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/archive/refs/tags/v1beta2-1.2.2-3.0.0.tar.gz"
```

Comment:
@carlocab FYI this caused brew audit to fail since the api version of the CRD is beta. Don't think i'll be able to get around this 

```
sparkctl:
  * Stable version URLs should not contain beta
Error: 1 problem in 1 formula detected
```

Reply:
Try adding this to `audit_exceptions/unstable_allowlist.json`.

We don't ship beta software in Homebrew, though, so we'll need to address whether the API version being in beta applies to the software itself, or just the API it interacts with that can be viewed as distinct from the software.

Reply:
i was under the impression that the project itself was considered stable but after reading though some of the docs it looks like it's still considered beta as well. We can close this out and I'll create a tap for it elsewhere. I apologize for not realizing that sooner.

Reply:
Sure, no problem. You're welcome to submit this again to Homebrew/core when it's no longer in beta.